gr-soapy/python/soapy/bindings/soapy_common.cc

```diff
@@ -49,22 +51,20 @@ static inline T string_to_setting(const std::string& str)
 }
 
 // Copied from SoapySDR 0.8
+//! convert empty and "false" strings to false, integers to their truthness
 template <>
 inline bool string_to_setting<bool>(const std::string& str)
 {
-    if (str == SOAPY_SDR_TRUE)
-        return true;
-    if (str == SOAPY_SDR_FALSE)
-        return false;
-
-    // zeros and empty strings are false
-    if (str == "0")
-        return false;
-    if (str == "0.0")
-        return false;
-    if (str == "")
+    if (str.empty() || str == SOAPY_SDR_FALSE) {
```

Comment:
What about lower-case version of `str`?

Reply:
was considering that, too, but I don't think we need to be more flexible there – `False` might be tempting.

If I were to change that behaviour however, I'd rather do it upstream in SoapySDR and then do the same in GR than the other way around